### ISSUE TITLE
[SDK 2.0.0-rc2] GET BillingAgreement misses Plan ID

### ISSUE THREAD

**Issue by kanadaj** [paypal/PayPal-NET-SDK#415]
### General information

* SDK/Library version: .NET Core 2.2
* Environment: Production
* `PayPal-Debug-ID` values: N/A
* Language, language version, and OS: .NET Core 2.2, Windows 10 1903 and CentOS 7.6

### Issue description

The documentation of the BillingAgreement GET request on paypal.com is a bit misleading - there is no actual Id included in the Plan element, so the RC version of the SDK cannot populate the `Id` field of the `PlanWithId Plan` property of an `Agreement`. Do note that while this is an upstream issue, the current stable version parses the plan body (since that is returned by the API) thus it was less of a problem before.

In short: https://developer.paypal.com/docs/api/payments.billing-agreements/v1/#billing-agreements_get returns a `Plan` object _without_ it's ID. The 2.0.0-rc2 expects _only_ an Plan ID.

**Comment by kanadaj** [comment]
Related issue:
https://github.com/paypal/PayPal-REST-API-issues/issues/92

**Comment by kanadaj** [comment]
@jaypatel512

**Comment by sucithrashanu** [comment]
@kanadaj 

Currently we do not have any stipulated timeline for the v2.0.0 release, but it may be available in near future.

Please Check [PayPal Community ](https://www.paypal-community.com)and  [MTS]( https://paypal.com/mts) 

Thanks!
